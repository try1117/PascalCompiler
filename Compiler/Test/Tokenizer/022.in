var
Var
VAr
VAR
vAr
