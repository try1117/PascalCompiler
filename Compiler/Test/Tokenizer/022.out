1  | 1  | KEYWORD_VAR              | var                      | 
2  | 1  | KEYWORD_VAR              | Var                      | 
3  | 1  | KEYWORD_VAR              | VAr                      | 
4  | 1  | KEYWORD_VAR              | VAR                      | 
5  | 1  | KEYWORD_VAR              | vAr                      | 
